Wrappers.h

```diff
@@ -4,35 +4,35 @@
 
 struct ShoeWrapper
 {
-    ShoeWrapper(Shoe* shoePtr) : theShoe(shoePtr) { }
-    ~ShoeWrapper() { delete theShoe; }
+    ShoeWrapper(Shoe* shoePtr);
+    ~ShoeWrapper();
     Shoe* theShoe = nullptr;
 };
 
 struct WurlitzerWrapper
 {
-    WurlitzerWrapper(Wurlitzer* wurlitzerPtr) : theWurli(wurlitzerPtr) { }
-    ~WurlitzerWrapper() { delete theWurli; }
+    WurlitzerWrapper(Wurlitzer* wurlitzerPtr);
+    ~WurlitzerWrapper();
     Wurlitzer* theWurli = nullptr;
 };
 
 struct BassWrapper
 {
-    BassWrapper(ElectricBass* bassPtr) : theBass(bassPtr) { }
-    ~BassWrapper() { delete theBass; }
+    BassWrapper(ElectricBass* bassPtr);
+    ~BassWrapper();
     ElectricBass* theBass = nullptr;
 };
 
 struct BandWrapper
 {
-    BandWrapper(Band* bandPtr) : theBand(bandPtr) { }
-    ~BandWrapper() { delete theBand; }
+    BandWrapper(Band* bandPtr);
+    ~BandWrapper();
     Band* theBand = nullptr;
 };
 
 struct ClothingWrapper
 {
-    ClothingWrapper(Clothing* clothesPtr) : theClothes(clothesPtr) { }
-    ~ClothingWrapper() { delete theClothes; }
+    ClothingWrapper(Clothing* clothesPtr);
```

Comment:
You aren't using this pointer member variable in this source file so you don't need to include the header for it.  
Forward declare the type in the header.
Include in the cpp
- [ ] revise